crates/net/discv4/Cargo.toml

```diff
@@ -31,7 +31,7 @@ secp256k1 = { workspace = true, features = [
 enr.workspace = true
 
 # async/futures
-tokio = { workspace = true, features = ["io-util", "net", "time"] }
+tokio = { workspace = true, features = ["io-util", "net", "time", "rt-multi-thread"] }
```

Comment:
do we need this?

Reply:
probably not, left over from something being messed up with my setup, for some reason I can't test discv4 without it